script/client

```diff
@@ -75,7 +76,7 @@ sub usage($) {
     }
 }
 
-GetOptions(\%options,"host=s","apibase=s","verbose|v","apikey:s","apisecret:s","help|?",) or usage(1);
+GetOptions(\%options,"host=s","apibase=s","verbose|v","apikey:s","apisecret:s", "params=s", "help|?",) or usage(1);
 
```

Comment:
shouldn't the --help (btw, -h is documented as alias for --help in the --help output, but gives: Option h is ambiguous (help, host)) give an hint ?